apps/schedules/lib/hours_of_operation.ex

```diff
@@ -146,6 +199,52 @@ defmodule Schedules.HoursOfOperation do
     {:error, :timeout}
   end
 
+  defp special_service_departures({:error, _} = error, _, _) do
+    error
+  end
+
+  defp special_service_departures(special_service_responses, headsigns, description) do
+    with {:ok, special_service_depature_maps} <-
+           special_service_departures_parser(special_service_responses, headsigns, description) do
+      {
+        :ok,
+        special_service_depature_maps
+      }
+    end
```

Comment:
The goal of the `with` is to make sure that any errors with the special service data are passed up the tree.
If we get an array that isn't an even length the function `special_service_depatures_parser/3` will return an error.  I believe removing the with call would cause this error to be lost/hidden.